src/ctree-app/ctree-landing.html

```diff
@@ -246,31 +376,55 @@
           <p>W</p>
           <p>W</p>
           <p>W</p>
-        </div><div id="participate" style="display: inline-block; width: 100vw; background-color: #FFFF00">
-          <section class="join">
-            <h2>Participate</h2>
-            <p>
-              <span class="bold">Join the open source project</span>
-              <br />
-              Get course credit or professional grade experience working on a
-              large scale open source project. Be part of a team, working
-              closely with other programmers and designers to complete a part of
-              a project. All regular contributors can add the project to their
-              LinkedIn resume and request a recommendation based on their
-              completed work. Go to
-              <span class="bold">dev.ctree.org</span> to get started.
-            </p>
-            <p>
-              <span class="bold">Test the project</span>
-              <br />
-              As the project develops we need real world examples of how it can
-              be used. If you want to use it with your nonprofit, online
-              community, or other organization please email
-              <span class="bold">contact@innovativeFuture</span>.
-            </p>
-          </section>
+          <p>W</p>
+          <p>W</p>
+          <p>W</p>
+        </div>
+        <div id="participate" class="section-styling">
+            <div class= "center-div">
+              <h1 align="center">Participate</h1>
+                <p>
+                  <div class="bold-text">Join the open source project</div>
+                  Get course credit or professional grade experience working on a large open source
+                  project.  Be part of a project team, working closely with our programmers and
+                  designers to complete a part of the project.  All regular contributors can add the
+                  project to their LinkedIn resume and request a recommendation based on their comleted
+                  work.  Go to <a href="dev.ctree.org">dev.ctree.org</a> to get started.
+                  </p>
+                  <p>
+                    <div class="bold-text">Test the project</div>
+                    As the project develops we need real world examples of how it can be used.  If you
+                    want to use it with your nonprofit, online community, or other organization Please
+                    email <a href = "mailto: contact@innovativeFuture.org">contact@innovativeFuture.org</a>
+                    </p>
+            </div>
```

Comment:
looks like the indentation for this code needs to be improved

Reply:
Done and pushed.

Reply:
Looks like there's still an extra indent for everything in the participate div, but at least it's consistent so it's not a big enough issue to block the PR.